spinach/brokers/base.py

```diff
@@ -85,6 +85,15 @@ def _to_namespaced(self, value: str) -> str:
     def register_periodic_tasks(self, tasks: Iterable[Task]):
         """Register tasks that need to be scheduled periodically."""
 
+    def set_concurrency_keys(self, tasks: Iterable[Task]):
```

Comment:
Could you add the `@abstractmethod` decorator please? It acts mostly as documentation.

Reply:
I didn't want to force new brokers to implement this method, but given the likelihood of new brokers, it's probably fine :)